pycypher/src/setup.py

```diff
@@ -51,7 +51,7 @@ def read_version():
     description=description,
     long_description=long_description,
     license='Apache License 2.0',
-    url=u'https://github.com/cleishm/libcypher-parser/tree/master/pycypher',
+    url=u'https://github.com/cleishm/libcypher-parser/tree/master',
```

Comment:
This link is shown on PyPI and users looking for documentation on how to use python bindings will click it.
Please explain why you think it is better to show documentation of underlying C library to users interested in using a Python library?

Reply:
I'm happy to add something about the python bindings into the top level README. The README in the subtree seemed to be more oriented at the usage of pycypher rather than at building the source tree?

Reply:
What is wrong with readme in subtree being oriented at usage? This is where users should be directed to from PyPI page.

Reply:
If that URL is where users will find documentation of how to use pycypher, that's certainly different. In which case, I agree that it should point to https://github.com/cleishm/libcypher-parser/tree/master/pycypher/doc. Unless you want to publish a webpage? You are welcome to create a subpage on http://cleishm.github.io/libcypher-parser.

Reply:
I'm glad you agree with it. Regarding webpage - maybe later.